service/src/main/java/bio/terra/workspace/service/resource/WsmResourceType.java

```diff
@@ -88,7 +89,13 @@
       "AZURE_STORAGE_ACCOUNT",
       ApiResourceType.AZURE_STORAGE_ACCOUNT,
       null,
-      ControlledAzureStorageResource.class);
+      ControlledAzureStorageResource.class),
+  GIT_REPO(
+      CloudPlatform.MULTI,
```

Comment:
We might think of backing this property out of this enum, since there are non-cloud-specific resources now. Or we can make it optional.

Reply:
added a TODO. I tried making it optional and it requires a lot more change in the ResourceDao so I think it should be in a separate PR.